src/components/Explore/panels/TraceViewPanelScene.tsx

```diff
@@ -47,6 +87,23 @@ export class TraceViewPanelScene extends SceneObjectBase<TracePanelState> {
                 component: SkeletonComponent,
               }),
             });
+          } else if (data.data?.state === LoadingState.Error) {
+            this.getTraceErrorMessage(data.data?.error, this.state.traceId)
```

Comment:
Error is deprecated

<img width="797" height="289" alt="Screenshot 2025-09-24 at 13 21 12" src="https://github.com/user-attachments/assets/8e0530e8-fdd9-4359-a735-2e293e3fbe7c" />